### ISSUE TITLE
SfPdfViewer change orientation

### ISSUE THREAD

**Issue by whitewabbit** [syncfusion/xamarin-demos#11]
when pdf viewer change orientation, the view will turn to white and if i change it back it will not respond and will give me this on my output:

06-14 15:58:20.201 D/Mono    (18571): Assembly Ref addref Syncfusion.SfPdfViewer.XForms.Android[0xb8931198] -> System.Core[0xb8a23768]: 14
The thread 0x12 has exited with code 0 (0x0).
06-14 15:58:21.151 I/art     (18571): Starting a blocking GC Explicit
06-14 15:58:21.181 I/art     (18571): Explicit concurrent mark sweep GC freed 11356(555KB) AllocSpace objects, 24(15MB) LOS objects, 40% free, 7MB/12MB, paused 707us total 38.267ms
06-14 15:58:21.191 D/Mono    (18571): GC_TAR_BRIDGE bridges 321 objects 31873 opaque 12451 colors 321 colors-bridged 321 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 1.14ms
06-14 15:58:21.191 D/Mono    (18571): GC_BRIDGE: Complete, was running for 43.97ms
06-14 15:58:21.191 D/Mono    (18571): GC_MINOR: (Nursery full) time 36.05ms, stw 36.70ms promoted 261K major size: 5872K in use: 4730K los size: 18432K in use: 17494K
06-14 15:58:22.931 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:22.931 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.12ms
06-14 15:58:22.931 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.84ms, stw 5.40ms promoted 36K major size: 5920K in use: 4766K los size: 18432K in use: 17510K
06-14 15:58:24.691 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:24.691 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:24.691 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.57ms, stw 5.16ms promoted 36K major size: 5952K in use: 4803K los size: 18432K in use: 17542K
06-14 15:58:26.931 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:26.931 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:26.931 D/Mono    (18571): GC_MINOR: (Nursery full) time 5.43ms, stw 6.44ms promoted 36K major size: 5984K in use: 4839K los size: 18432K in use: 17542K
06-14 15:58:29.061 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:29.061 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:29.061 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.79ms, stw 5.55ms promoted 36K major size: 6032K in use: 4875K los size: 19456K in use: 17606K
06-14 15:58:30.831 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:30.831 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:30.831 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.54ms, stw 5.13ms promoted 36K major size: 6064K in use: 4912K los size: 19456K in use: 17606K
06-14 15:58:32.671 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:32.671 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:32.671 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.53ms, stw 5.05ms promoted 36K major size: 6096K in use: 4948K los size: 19456K in use: 17606K
06-14 15:58:34.451 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:34.451 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:34.451 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.56ms, stw 5.10ms promoted 36K major size: 6144K in use: 4985K los size: 19456K in use: 17734K
06-14 15:58:36.281 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:36.281 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:36.281 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.89ms, stw 5.44ms promoted 36K major size: 6176K in use: 5021K los size: 19456K in use: 17734K
06-14 15:58:38.071 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:38.071 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.12ms
06-14 15:58:38.071 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.59ms, stw 5.16ms promoted 36K major size: 6208K in use: 5057K los size: 19456K in use: 17734K
06-14 15:58:40.211 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:40.211 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:40.211 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.54ms, stw 5.08ms promoted 36K major size: 6240K in use: 5094K los size: 19456K in use: 17734K
06-14 15:58:41.991 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 321 xref 0 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.08ms tarjan 25.24ms scc-setup 0.18ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:41.991 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:41.991 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.59ms, stw 5.14ms promoted 36K major size: 6288K in use: 5130K los size: 19456K in use: 17734K
06-14 15:58:44.021 I/art     (18571): Starting a blocking GC Explicit
06-14 15:58:44.061 I/art     (18571): Explicit concurrent mark sweep GC freed 24802(1409KB) AllocSpace objects, 0(0B) LOS objects, 39% free, 7MB/12MB, paused 796us total 42.660ms
06-14 15:58:44.071 D/Mono    (18571): GC_TAR_BRIDGE bridges 397 objects 435 opaque 11 colors 397 colors-bridged 397 colors-visible 397 xref 1 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.13ms tarjan 0.66ms scc-setup 0.21ms gather-xref 0.02ms xref-setup 0.00ms cleanup 2.58ms
06-14 15:58:44.071 D/Mono    (18571): GC_BRIDGE: Complete, was running for 53.47ms
06-14 15:58:44.071 D/Mono    (18571): GC_MAJOR: (Minor allowance) time 36.25ms, stw 37.17ms los size: 15360K in use: 3258K
06-14 15:58:44.071 D/Mono    (18571): GC_MAJOR_SWEEP: major size: 5824K in use: 4388K
06-14 15:58:46.051 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 397 xref 1 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.13ms tarjan 0.66ms scc-setup 0.21ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:46.051 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:46.051 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.70ms, stw 6.35ms promoted 36K major size: 5824K in use: 4424K los size: 15360K in use: 3258K
06-14 15:58:48.121 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 397 xref 1 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.13ms tarjan 0.66ms scc-setup 0.21ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:48.121 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.12ms
06-14 15:58:48.121 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.35ms, stw 4.91ms promoted 36K major size: 5872K in use: 4461K los size: 15360K in use: 3514K
06-14 15:58:49.911 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 397 xref 1 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.13ms tarjan 0.66ms scc-setup 0.21ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:49.911 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.10ms
06-14 15:58:49.911 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.43ms, stw 4.96ms promoted 36K major size: 5904K in use: 4497K los size: 15360K in use: 3514K
06-14 15:58:51.811 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 397 xref 1 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.13ms tarjan 0.66ms scc-setup 0.21ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:51.811 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.12ms
06-14 15:58:51.811 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.31ms, stw 4.88ms promoted 36K major size: 5936K in use: 4533K los size: 15360K in use: 3514K
06-14 15:58:53.601 D/Mono    (18571): GC_TAR_BRIDGE bridges 0 objects 0 opaque 0 colors 0 colors-bridged 0 colors-visible 397 xref 1 cache-hit 0 cache-semihit 0 cache-miss 0 setup 0.13ms tarjan 0.66ms scc-setup 0.21ms gather-xref 0.02ms xref-setup 0.00ms cleanup 0.00ms
06-14 15:58:53.601 D/Mono    (18571): GC_BRIDGE: Complete, was running for 0.11ms
06-14 15:58:53.601 D/Mono    (18571): GC_MINOR: (Nursery full) time 4.31ms, stw 4.87ms promoted 36K major size: 5984K in use: 4570K los size: 15360K in use: 3514K

**Comment by Sathish6** [comment]
Hi White, 

Greetings from Syncfusion. 

We tried to reproduce the reported issue in our side, but the PDF document is displayed properly in SfPdfViewer in Xamarin.Forms(Android) when the device orientation changes. We have used PdfViewer sample available in the below link. 
 
https://github.com/syncfusion/xamarin-demos/tree/master/Forms 
 
Kindly provide the following details to analyze more on this issue. 
 
1.	Android device in which you are facing the issue.
2.	SfPdfViewer version
3.	Xamarin.Forms version.
  
Regards,  
Sathish  
(I work for Syncfusion Software)
